.ci/operatorhub.sh

```diff
@@ -0,0 +1,114 @@
+#!/bin/bash
+
+if [ -z ${COMMUNITY_OPERATORS_REPOSITORY} ]; then
+    COMMUNITY_OPERATORS_REPOSITORY="$(dirname $(dirname $(pwd)))/operator-framework/community-operators"
+    echo "COMMUNITY_OPERATORS_REPOSITORY not set, using ${COMMUNITY_OPERATORS_REPOSITORY}"
+fi
+
+if [ ! -d ${COMMUNITY_OPERATORS_REPOSITORY} ]; then
+    echo "${COMMUNITY_OPERATORS_REPOSITORY} doesn't exist, aborting."
+    exit 1
+fi
+
+OLD_PWD=$(pwd)
+VERSION=$(grep -v \# jaeger-operator.version)
```

Comment:
Was surprised to see this `jaeger-operator.version` file appear. Is it instead of `jaeger.version` or as well as? Is the rename due to the csv label or something? Thought we were aligning the operator and jaeger components version so not sure why we need separate files?

Reply:
It's in addition to the existing version file. We could combine both in one file, but it would make the command a bit more complex (`grep | awk` instead of simply `grep`). Would it be OK if I do it right after merging this?

Reply:
I implemented this change already.